plugins/reolink/src/reolink-api.ts

```diff
@@ -74,13 +74,13 @@ export class ReolinkCameraClient {
     }
 
     async login() {
-        if (this.tokenLease > Date.now()) {
+        if ((!this.forceToken ? true : this.tokenLease !== Infinity) && this.tokenLease > Date.now()) {
```

Comment:
this change is not necessary if forceToken is readonly

Reply:
done